pkg/services/org/orgimpl/store_test.go

```diff
@@ -773,7 +773,7 @@ func TestIntegration_SQLStore_GetOrgUsers_PopulatesCorrectly(t *testing.T) {
 	assert.Equal(t, int64(1), actual.UserID)
 	assert.Equal(t, "viewer@localhost", actual.Email)
 	assert.Equal(t, "Viewer Localhost", actual.Name)
-	assert.Equal(t, "Viewer", actual.Login)
+	assert.Equal(t, "viewer", actual.Login)
```

Comment:
I don't think this change is required when enforcing this from the store right?

Reply:
I ran the test and it needs to be changed. At least from the change we have made in this PR
```
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-Viewer
        	            	+viewer
```

Reply:
There is another test failing as well